pkgs/applications/editors/micro/default.nix

```diff
@@ -34,6 +34,15 @@ buildGoModule rec {
     install -Dm644 assets/micro-logo-mark.svg $out/share/icons/hicolor/scalable/apps/micro.svg
   '';
 
+  fixupPhase = ''
```

Comment:
instead of overriding fixupPhase, you can set postFixup ( or preFixup not sure which one is better )